lib/Partitioner/Partitioner.cpp

```diff
@@ -26,6 +26,13 @@
 using namespace glow;
 using llvm::isa;
 
+// Sorted the std::pair<DAGNode *, uint64_t> based on the second from min to
+// max.
+auto sortMinMemory(const std::pair<Function *, uint64_t> &a,
```

Comment:
Is there  a reason for using the `auto foo() -> bool` syntax instead of simply `bool foo()`?  It feels kind of distracting to use this style here as a one-off.

Reply:
Good catch! Initially, I noticed the closure and asked @beicy to change it to a function. But I haven't noticed that she forgot to change the return type while changing it ;-)

Reply:
Thanks! I didn't notice that :P Will add a follow up PR.